zmon-common/src/main/java/org/zalando/zmon/domain/Feedback.java

```diff
@@ -0,0 +1,14 @@
+package org.zalando.zmon.domain;
+
+public class Feedback {
```

Comment:
Well, technically you're not handling feedback itself in this class, right? I'd change it to something like `FeedbackMetadata` or alike...